bip-0039.mediawiki

```diff
@@ -156,6 +156,9 @@ JavaScript:
 * https://github.com/bitpay/bitcore-mnemonic
 * https://github.com/bitcoinjs/bip39 (used by [[https://github.com/blockchain/My-Wallet-V3/blob/v3.8.0/src/hd-wallet.js#L121-L146|blockchain.info]])
```

Comment:
O

Reply:
@Jakkon1981Boonmark Your nonsense comments are not welcome.  I have reported.